docs/index.html

```diff
@@ -7,7 +7,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=9;IE=10;IE=Edge,chrome=1"/>
 
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-	<link href="docs.bundle.css" rel="stylesheet">
+	<link href="http://nucleus.dev/docs/docs.bundle.css" rel="stylesheet">
```

Comment:
Doesn't this introduce problems with the docs on gh pages?

Reply:
Oooh this is a mistake :P